modules/objectdb_profiler/editor/objectdb_profiler_panel.cpp

```diff
@@ -0,0 +1,447 @@
+	snapshot_list->set_columns(1);
+	snapshot_list->set_column_titles_visible(true);
+	snapshot_list->set_column_title(0, "Snapshots");
+	snapshot_list->set_column_expand(0, true);
+	snapshot_list->set_column_clip_content(0, true);
+	snapshot_list->connect("item_selected", callable_mp(this, &ObjectDBProfilerPanel::_show_selected_snapshot));
+	snapshot_list->connect("item_edited", callable_mp(this, &ObjectDBProfilerPanel::_edit_snapshot_name));
+	snapshot_list->set_h_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+	snapshot_list->set_v_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+	snapshot_list->set_anchors_preset(LayoutPreset::PRESET_FULL_RECT);
+
+	snapshot_list->set_allow_rmb_select(true);
+	snapshot_list->connect(SNAME("item_mouse_selected"), callable_mp(this, &ObjectDBProfilerPanel::_snapshot_rmb));
+
+	rmb_menu = memnew(PopupMenu);
+	add_child(rmb_menu);
+	rmb_menu->connect(SceneStringName(id_pressed), callable_mp(this, &ObjectDBProfilerPanel::_rmb_menu_pressed).bind(false));
+
+	HBoxContainer *diff_button_and_label = memnew(HBoxContainer);
+	diff_button_and_label->set_h_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+	snapshot_column->add_child(diff_button_and_label);
+	Label *diff_against = memnew(Label(TTR("Diff Against:")));
+	diff_button_and_label->add_child(diff_against);
+
+	diff_button = memnew(OptionButton);
+	diff_button->set_h_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+	diff_button->connect("item_selected", callable_mp(this, &ObjectDBProfilerPanel::_apply_diff));
+	diff_button_and_label->add_child(diff_button);
+
+	// tabs of various views right for each snapshot
+	view_tabs = memnew(TabContainer);
+	root_container->add_child(view_tabs);
+	view_tabs->set_custom_minimum_size(Size2(300 * EDSCALE, 0));
+	view_tabs->set_v_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+	view_tabs->connect("tab_changed", callable_mp(this, &ObjectDBProfilerPanel::_view_tab_changed));
+
+	add_view(memnew(SnapshotSummaryView));
+	add_view(memnew(SnapshotClassView));
+	add_view(memnew(SnapshotObjectView));
+	add_view(memnew(SnapshotNodeView));
+	add_view(memnew(SnapshotRefCountedView));
+	add_view(memnew(SnapshotJsonView));
+
+	set_enabled(false);
+
+	// load all the snapshot names from disk
+	Ref<DirAccess> snapshot_dir = _get_and_create_snapshot_storage_dir();
+	TreeItem *last_snapshot_button = nullptr;
+	if (!snapshot_dir.is_null()) {
+		for (const String &file_name : snapshot_dir->get_files()) {
+			Vector<String> name_parts = file_name.split(".");
+			if (name_parts.size() != 2 || name_parts[1] != "odb_snapshot") {
+				ERR_PRINT("ObjectDB Snapshot file did not have .odb_snapshot extension. Skipping: " + file_name);
+				continue;
+			}
+
+			last_snapshot_button = _add_snapshot_button(name_parts[0], snapshot_dir->get_current_dir().path_join(file_name));
+		}
+		// simulate clicking on the last snapshot we loaded from disk
+		if (last_snapshot_button != nullptr) {
+			snapshot_list->set_selected(last_snapshot_button);
+			snapshot_list->ensure_cursor_is_visible();
+		}
```

Comment:
When testing on a non-trivial project I'm seeing hundreds/thousands of `Resource file not found` errors when starting the editor while having existing snapshots on disk.

As far as I can tell it's coming from this piece of code, as it causes the resource loading in `EditorDebuggerRemoteObject::update_props` to happen fairly early in the program flow.

When omitting this simulated clicking and simply clicking yourself instead you see no such errors.